src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java

```diff
@@ -685,6 +677,15 @@ void completeFlow() {
         testPage.clickElementById("preparesFood-you");
         testPage.clickContinue();
 */
+        testPage.navigateToFlowScreen("mdBenefitsFlow/applicantApplying");
```

Comment:
The code looks good, approved, but I do wonder if we should move this logic into a helper method for the test? Something like completeApplicantInformationSection() that does all this logic around different potential paths / conditional logic tree navigation?

Reply:
that's a good point. For now, I added a fork in the testing so that we could explore an individual flow and a family flow.  Let's discuss tomorrow in standup and I can shift in a follow up PR.